tests/test_op.py

```diff
@@ -1171,9 +1177,15 @@ def test_drop_table_add_kw(self):
 
     def test_create_table(self):
         schema_obj = schemaobj.SchemaObjects()
-        table = schema_obj.table("x", Column("q", Integer))
+        table = schema_obj.table("x",
+                                 Column("q", Integer),
+                                 postgresql_partition_by='x',
+                                 info={'custom': 'value'},
+                                 comment='some comment')
         op = ops.CreateTableOp.from_table(table)
         is_not_(op.to_table(), table)
+        eq_(op.to_table().comment, table.comment)
+        eq_(op.to_table().info, table.info)
 
```

Comment:
**mike bayer** (zzzeek) wrote:

see if we can add "prefixes"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2989